packages/api/src/auth/index.ts

```diff
@@ -77,16 +99,39 @@ export const getAuthenticationContext = async ({
   event: APIGatewayProxyEvent | Request
   context: LambdaContext
 }): Promise<undefined | AuthContextPayload> => {
-  const type = getAuthProviderHeader(event)
+  const cookieHeader = parseAuthorizationCookie(event)
+  const typeFromHeader = getAuthProviderHeader(event)
 
-  // No `auth-provider` header means that the user is logged out,
-  // and none of this auth malarky is required.
-  if (!type) {
+  // Short-circuit - if no auth-provider or cookie header, its
+  // an unauthenticated request
+  if (!typeFromHeader && !cookieHeader) {
     return undefined
   }
 
-  const { schema, token } = parseAuthorizationHeader(event)
+  let token: string | undefined
+  let type: string | undefined
+  let schema: string | undefined
+
+  // The actual session parsing is done by the auth decoder
+  // Priority given to cookie
+  if (cookieHeader) {
+    token = cookieHeader.rawCookie
+    type = cookieHeader.type
+    schema = 'cookie'
```

Comment:
We're leaving as is, because it's confusing anyway. 

Ideally at some point we will:
- rename `schema` -> `scheme`
- rename `token` -> `cookie(s)`